store/sqlstore/session_store.go

```diff
@@ -102,18 +97,16 @@ func (me SqlSessionStore) Get(sessionIdOrToken string) (*model.Session, *model.A
 func (me SqlSessionStore) GetSessions(userId string) ([]*model.Session, *model.AppError) {
 	var sessions []*model.Session
 
-	tcs := me.Team().GetTeamsForUser(userId)
```

Comment:
Here is the same, we call it asynchronously to make the query concurrently with the following select. I would expect the explicit async approach here.